pfi.c

```diff
@@ -116,11 +116,19 @@ static int __init pfi_init(void)
         printk(KERN_ALERT DRV_NAME ": failed to get file descriptor\n");
         cleanup_sysfs();
         return -1;
-    } 
+    }
 
     result = gpio_request_one(pin, GPIOF_IN, DRV_NAME " irq");
     if (result >= 0)
     {
+        // Set debounce to 500 microseconds. This actually sets it for the entire
+        // gpio bank, not just the specific pin.
+        result = gpio_set_debounce(pin, 500);
+        if(result != 0)
+        {
+            printk(KERN_ALERT DRV_NAME ": gpi_set_debounce returned %d", result);
```

Comment:
Could add a `\n`, I guess.

Reply:
I guess it could. Funny, I didn't really notice anything wrong with it in /var/log/messages.

Reply:
See https://github.com/Vorne/pfi/pull/5/commits/32939cb76e9cc12989466e9b6471c79bdb437360. The module was also rebuilt.